README.md

```diff
@@ -308,6 +308,10 @@ spec:
   ratioMetric: # either thresholdMetric or ratioMetric must be provided
     counter: true | false # true if the metric is a monotonically increasing counter,
                           # or false, if it is a single number that can arbitrarily go up or down
+                          # ignored when using "raw"
+    type: success | failure # required with "raw", indicates how the stored ratio was calculated:
```

Comment:
Would naming it `raw_type` make the usage clearer?

Reply:
I'm not a fan of the increased verbosity, defining something with OpenSLO already takes quite a lot more typing than with pyrra or sloth.

This `type` isn't specific to the raw ratio metric because when defining good+total or bad+total, since those also have this type. The difference is the type in those cases is implied, since it can be deducted from the definition, while with `raw` it can't and must be provided explicitly.

(Also, I believe it'd be `rawType` to be in line with the rest of the spec.)

Reply:
While `type` is also present on good/bad/total, it's a very different meaning. Currently, `type` on the `MetricSource` for good/bad/total is defined as the source type, with examples of "Datadog" and "Prometheus".

To me, that's very different from a `type` which must be one of "success" or "failure". I fear altering the meaning of `type` based on the object it is attached to will make it more confusing.

You're correct, I should've referred to it as `rawType`. I would not agree that typing an extra three characters adds significant verbosity to the spec.

While I agree OpenSLO is more verbose than pyrra and sloth, the intent for OpenSLO is much broader than the other tools which are Prometheus specific. As such, it will naturally lead to more verbosity due to the necessary generalizations.

Reply:
Switched it to `rawType`.